app/assets/javascripts/admin/addon/components/dashboard-new-feature-item.gjs

```diff
@@ -1,50 +1,113 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import { service } from "@ember/service";
 import { and, not } from "truth-helpers";
 import CookText from "discourse/components/cook-text";
+import DToggleSwitch from "discourse/components/d-toggle-switch";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 import i18n from "discourse-common/helpers/i18n";
+import { bind } from "discourse-common/utils/decorators";
+import DTooltip from "float-kit/components/d-tooltip";
 
-const DashboardNewFeatureItem = <template>
-  <div class="admin-new-feature-item">
-    <div class="admin-new-feature-item__content">
-      <div class="admin-new-feature-item__header">
-        {{#if (and @item.emoji (not @item.screenshot_url))}}
-          <div class="admin-new-feature-item__new-feature-emoji">
-            {{@item.emoji}}
-          </div>
-        {{/if}}
-        <h3>
-          {{@item.title}}
-        </h3>
-        {{#if @item.discourse_version}}
-          <div class="admin-new-feature-item__new-feature-version">
-            {{@item.discourse_version}}
-          </div>
-        {{/if}}
-      </div>
+export default class DiscourseNewFeatureItem extends Component {
+  @service siteSettings;
+  @tracked enabled;
+
+  @bind
+  initEnabled() {
+    this.enabled = this.siteSettings[this.args.item.experiment_setting];
+  }
 
-      {{#if @item.screenshot_url}}
-        <img
-          src={{@item.screenshot_url}}
-          class="admin-new-feature-item__screenshot"
-          alt={{@item.title}}
-        />
-      {{/if}}
+  @action
+  async toggle() {
+    this.enabled = !this.enabled;
+    try {
+      await ajax("/admin/toggle-feature", {
+        type: "POST",
+        data: {
+          setting_name: this.args.item.experiment_setting,
+        },
+      });
```

Comment:
We should do a toast to say the feature has been enabled, but the user may need to wait a short time for it to become available.

Also, I think it's worth disabling the toggle for say 3-5 seconds after the experiment has been toggled. This will introduce a nice throttle to stop people toggling back and forth and doing unnecessary work.

It would also be best practice for any plugins using the site setting change check in hosted site to maybe enqueue a job to do further changes when a feature is toggled this way.